elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -92,3 +113,34 @@ $LOCAL-VARIABLES: (
 :host([hidden]) {
   display: none;
 }
+
+/// ===========================================================================
+/// CONTEXTS: DARK & SATURATED
+/// ===========================================================================
+/// In dark & saturated contexts, we override the local color vars
+
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="dark"]) {
```

Comment:
This might be a little over complicated for what we're doing here.  Instead you can do: `:host([pfe-variant="earth"][on="dark"]) {`

Reply:
I don't disagree, the file was like this when I got here. I thought it was from a pair-programming knowledge share from you & Adam --  I didn't know if there was some advantage? I also looked into creating a mixin that would help us to nest selectors but I don't think it can be done since they all have to sit inside the parenthesis on the host right?

like this:
```scss
:host {
  @include append([pfe-variant="wind"]) {
    @include append([on="dark"]) {
      color: white;
    }
  }
}
```
would ideally generate 
```
:host([pfe-variant="wind"][on="dark"]) {
      color: white;
}
```

Reply:
replaced @at-root with :host throughout the file for readability

Reply:
@starryeyez024 Did you push up those changes yet?